src/NServiceBus.Core/Pipeline/BehaviorContext.cs

```diff
@@ -19,6 +20,11 @@ public IBuilder Builder
             }
         }
 
+        public Task RaiseNotification<T>(T notification) where T : INotification
+        {
+            return Builder.Build<EventAggregator>().Raise(notification);
```

Comment:
seems a bit inefficient to always resolve the eventaggregator from the container?